modules/index/src/integrationTest/java/org/apache/ignite/internal/index/ItIndexNodeFinishedRwTransactionsCheckerTest.java

```diff
@@ -64,7 +64,7 @@ public class ItIndexNodeFinishedRwTransactionsCheckerTest extends ClusterPerClas
 
     private static final String TABLE_NAME = "TEST_TABLE";
 
-    private static String zoneNameForUpdateCatalogVersionOnly = "FAKE_TEST_ZONE";
+    private String zoneNameForUpdateCatalogVersionOnly = "FAKE_TEST_ZONE";
```

Comment:
Let's make it a constant and use `dropAllTables `+ `dropAllZonesExceptDefaultOne` in tearDown method

Reply:
Why not.